python/pyarrow/tests/test_scalars.py

```diff
@@ -555,17 +560,23 @@ def test_binary(value, ty, scalar_typ):
     assert str(s) == str(value)
     assert repr(value) in repr(s)
     assert s.as_py() == value
+    assert bytes(s) == value
     assert s != b'xxxxx'
 
     buf = s.as_buffer()
     assert isinstance(buf, pa.Buffer)
     assert buf.to_pybytes() == value
 
+    memview = memoryview(s)
+    assert isinstance(memview, memoryview)
```

Comment:
```suggestion
```

This line can be removed and some additional assertions can be added, like here:

https://github.com/apache/arrow/blob/60b5ab9ee0bf070f03cf5c92fe0add0257543dfd/python/pyarrow/tests/test_tensor.py#L217-L226

Reply:
Updated!